### PR TITLE
added controller

### PR DESCRIPTION
**Pull request by lizzymikhailovskaia**

### CODE REVIEW

File: app/models/location.rb [hunk 1]
```diff
@@ -0,0 +1,5 @@
+class Location < ActiveRecord::Base
+  belongs_to :trip
+  has_many :tasks
+  has_many :photos
```
Comment (vovanmix) [inline app/models/location.rb:4]: Has many comments as well?

**Comment by lizzymikhailovskaia** [comment]
Why?

**Review by vovanmix — CHANGES_REQUESTED** [review]
Good, but need to add some more associations

File: app/models/location.rb (hunk 1 above)
Comment (lizzymikhailovskaia) [reply app/models/location.rb:4]: added

Comment (lizzymikhailovskaia) [inline app/models/location.rb]: 'class Location < ActiveRecord::Base
  belongs_to :trip
  has_many :tasks
  has_many :photos
  has_many :comments
end
'

>

**Comment by lizzymikhailovskaia** [comment]
LGTM 👍

**Comment by lizzymikhailovskaia** [comment]
looks good to me

**Pull request merged**